tests/test_outline.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from outlines.outline import Outline
+
+
+def test_outline_int_output():
+    # Mock the model
+    model = MagicMock()
```

Comment:
I thought that using a real model here might result in this being a flaky test, WDYT?

Reply:
Yes, good idea. Plus it would add a lot of overhead for no obvious improvement.